docs/user_guide/psi_v2_benchmark.md

```diff
@@ -0,0 +1,421 @@
+# PSI V2 Benchmark
+
+This document will introduce the PSI V2 Benchmark. It uses the PSI V2 version of the interface.
+## Building from source
+
+```
+git clone https://github.com/secretflow/psi.git
+cd psi
+bazel build //... -c opt
+```
+
+If the building is successful, you will find an executable file named `main` in the 'bazel-bin/psi' directory. We will use `./main` file, combined with config file to run different PSI protocols. Such as:
+
+```
+./main --config configs/sender.config
+./main --config configs/receiver.config
+
+```
+## Generate Data
+
+
+In order to measure the performance of different PSI protocols under different data scales, we need to generate dummy data through [generate_psi.py](https://github.com/secretflow/secretflow/blob/main/docs/developer/benchmark/resources/generate_psi.py)
+
+```python
+python3 generate_psi.py 10000000 10000000
+
+python3 generate_psi.py 100000000 100000000
+```
+
+## Prepare config file
+
+We use the config file to specify different PSI protocols and input data.
+
+### sender.config
+```
+{
+         "psi_config": {
```

Comment:
Seems indent not aligned

Reply:
may be fix, please check again.

Reply:
may be fix, please check again. Thanks very much.